system-int-test/src/cmr/system_int_test/utils/ingest_util.clj

```diff
@@ -631,6 +632,11 @@
 
 ;;; fixture - each test to call this fixture
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(defn set-collection-ingest-umm-version-to-current
+  "Set the collection ingest-accept-umm-version to current-collection-version."
+  []
+  (side/eval-form `(common-config/set-collection-umm-version!
+                     umm-versioning/current-collection-version)))
```

Comment:
Wonder if we have the same issue with service and variable - i.e. their ingest tests are NOT testing against the latest umm version. If that's the case, we need to fix that too.

Reply:
Yes, good point. Right now we do not have a way to test the latest UMM version for variable/service similar to collection before this fix. We should do similar thing here to tie the ingest version to the latest UMM version supported internally by CMR.